src/components/ui/pagination/createRangeArray.ts

```diff
@@ -0,0 +1,9 @@
+export const createRangeArray = (start: number, end: number): number[] => {
```

Comment:
need move to utils or helpers folder in common

Reply:
Create new folder in common and move helper function